### PR TITLE
Updates to README

### PR DESCRIPTION
**Pull request by elizabethengelman**
- update configuration section
- removed creating and migrating the db as a prerequisite for running test - it doesn't seem to be necessary
- add a public.toml.example config file to be copied and updated locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,55 +2,55 @@
...
+      - Mac: `$HOME/Library/Ethereum`
+      - Linux: `$HOME/.ethereum`
+
+  - when using parity:
+    - The IPC file is called `jsonrpc.ipc`.
+    - The default location is:
+      - Mac: `$HOME/Library/Application\ Support/io.parity.ethereum/`
+      - Linux: `$HOME/.local/share/io.parity.ethereum/`
+
+- `environments/infura.toml` can be used to run commands with a full
```
Comment (rmulhol) [inline README.md]: I might rephrase as "run commands against infura, if local node is unavailable." You could technically point any of the config files at a full archive node if you had your local node configured in archive mode

Comment (elizabethengelman) [reply README.md]: Good call, updated in 194d81f

File: README.md [hunk 2]
```diff
@@ -2,55 +2,54 @@
...
+`go get github.com/vulcanize/vulcanizedb`
 
+## Setting up the Database
 1. Install Postgres
 1. Create a superuser for yourself and make sure `psql --list` works without prompting for a password.
-1. `createdb vulcanize_private`
+1. `createdb vulcanize_public`
 1. `cd $GOPATH/src/github.com/vulcanize/vulcanizedb`
-1. Import the schema: `psql vulcanize_private < db/schema.sql`
+1. Import the schema: `psql vulcanize_public < db/schema.sql`
```
Comment (gustin) [inline README.md]: If I do the import, then run migrations they fail. Wondering if that impacts the ability to run future migrations. Should we just recommend one way to setup the schema?

Comment (elizabethengelman) [reply README.md]: Good call! I'm leaning toward just recommending setting the schema up by running the migrations. It seems like the [migrate library](https://github.com/mattes/migrate) we're using handles tracking if a migration has been run yet or not.

Comment (elizabethengelman) [reply README.md]: cc93310

**Comment by AFDudley** [comment]
Looks like removing the migrations broke Travis? Maybe not at all? Sorry it's late. Since I'm the loose cannon here I'm happy to leave it broken until the morning/afternoon. :)

**Comment by elizabethengelman** [comment]
🤔 taking a look at the broken Travis build

**Pull request merged**